packages/@ember/-internals/runtime/lib/system/array_proxy.js

```diff
@@ -11,14 +11,14 @@ import {
   removeArrayObserver,
   replace,
   getChainTagsForKey,
-  tagForProperty,
+  CUSTOM_TAG_FOR,
```

Comment:
When did this API land? I just want to make sure it existed in 3.16...

Reply:
it landed with one of the more recent bugfixes, and I believe was backported.